### ISSUE TITLE
Support Stormpath's Oauth in Spring Security

### ISSUE THREAD

**Issue by mrioan** [stormpath/stormpath-sdk-java#211]

**Comment by omgitstom** [comment]
http://docs.stormpath.com/guides/token-management/

**Comment by george-hawkins-aa** [comment]
Hello @omgitstom and @mrioan - I got social login for Google and Facebook working with the Stormpath Spring Boot setup. I've put [a project](https://github.com/george-hawkins-aa/stormpath-spring-boot-example) up on Github that demonstrates this. For the TL;DR details on what I did see the [social login section](https://github.com/george-hawkins-aa/stormpath-spring-boot-example#social-login) of the README. My write-up is actually much longer than the actual code involved - which after a lot of hacking turned out not to be too bad.

![social login](https://cloud.githubusercontent.com/assets/12233059/11369390/e0a1ac0c-92be-11e5-989a-b2af01957036.png)
